.swiftlint.yml

```diff
@@ -3,7 +3,7 @@ custom_rules:
     included: ".*\\.swift"
     excluded: ".*Tests\\.swift"
     name: "Missing SwifterSwift: prefix"
-    regex: "///\\s(?!SwifterSwift:\\s)\\w+[^\\n]*$"
+    regex: "\\n\\n(\\s*///\\s(?!SwifterSwift:\\s)\\w+[^\\n]*$)"
```

Comment:
This change means that only the first line of documentation comments is checked. The parentheses indicate where the warning is highlighted in the code.